docs/architecture.md

```diff
@@ -9,9 +9,9 @@ _Some incomplete notes_
 * **Driver** - Interfaces with [Chrome Debugging Protocol](https://developer.chrome.com/devtools/docs/debugger-protocol)  ([API viewer](https://chromedevtools.github.io/debugger-protocol-viewer/))
 * **Gatherers** - Uses Driver to collect information about the page. Minimal post-processing.
   * **Artifacts** - output of a gatherer
-* **Audit** - Tests for a single feature/optimization/metric. Using the Artifacts as input, an audit evaluates a test and  resolves to a score which may be pass/fail/numeric. Formatting note: The meta description may contain markdown links and meta title may contain markdown code.
+* **Audit** - Tests for a single feature/optimization/metric. Using the Artifacts as input, an audit evaluates a test and resolves to a numeric score, described by scoreDisplayMode to be pass/fail or numeric. Formatting note: The meta description may contain markdown links and meta title may contain markdown code.
```

Comment:
maybe move the `scoreDisplayMode` comment to the "formatting note"? It's pretty tangential to the rest of the sentence.